src/listeners/messageCreate.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable operator-linebreak */
+/**
+ * "messageCreate" event listener for the bot.
+ */
+
+import { Client, Message } from 'discord.js';
+import {
+  extractNotionPageIdFromTreadByChannel,
+  getChatOpenAIPromptResponse,
+  getFormattedPrompt,
+  SUPPORT_IDENTIFIER,
+  updateNotionSupportTicketsDBEntry,
+} from '../utils';
+
+export default (client: Client): void => {
+  client.on('messageCreate', async (message: Message) => {
+    const msgChannel = await client?.channels
+      .fetch(message?.channelId)
+      .then((messageChannel: any) => {
+        console.log(messageChannel?.name);
+        return messageChannel;
+      })
+      .catch(console.error);
+
+    // Indicates that the community member is asking a software related question.
+    const containSupportIdentifier = message?.content?.includes('#question');
+    if (containSupportIdentifier) {
```

Comment:
need to inform the user if this condition fails

Reply:
Nope, we don't. This is handling any message that is created.

Reply:
If we dont find the identify then it means the user wasn't trying to get an open ai response